docs/user-guide/add-git-server.md

```diff
@@ -100,10 +100,10 @@ For [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/app-passwords
       </TabItem>
     </Tabs>
 
-3. Create a secret in the namespace where KubeRocketCI is installed (default is `edp`) to store the Git account credentials, including the **id_rsa**, **username**, and **token** fields.
+3. Create a secret in the namespace where KubeRocketCI is installed (default: `edp`) to securely store the Git account credentials, including the **id_rsa**, **username**, and **token** fields:
```

Comment:
can we swith to 'krci' namespace?

Reply:
Renaming "edp" to "krci" might confuse readers as we use "edp" in the rest of our documentation.